app/views/projects/projects/show.html.erb

```diff
@@ -1,21 +1,85 @@
-<h1><%= @project.title %></h1>
+<div class="row">
+<nav class="projects-nav">
+  <div class="projects-nav__title">
+    <h1><%= @project.title %></h1>
+  </div>
+  <div class="project-nav__menu">
+    <ul class="inline-list">
+      <li>
+        <a href="#">Projects home</a>
+      </li>
+      <li>
+        <a href="#">Tasks</a>
+      </li>
+      <li>
+        <a href="#">Message project owner</a>
+      </li>
+    </ul>
+  </div>
+</nav> 
+</div>     
 
-<div>
-  <%= sanitize @project.briefing %>
-</div>
+<div class="inner-canvas">
+  <div class="inner-canvas-header">
+   <div class="row">
+      <h1><%= @project.title %></h1>
+    </div>
+  </div>
 
-<aside>
-  <h1><%= @project.owner.name %></h1>
+  <div class="inner-canvas-body">
+    <div class="row">
+      <div class="project-body">
+        <div class="project-briefing">
+          <%= sanitize @project.briefing %>
+          <p class="project-meta">16 contributors</p>
+          <div class="project-workload">
+            <h3 class="project-workload-title">Workload</h3>
+            <div class="project-workload-diagram">
+              <progress max="100" value="56" class="project-workload-indicator"></progress><span class="project-workload-value">56%</span>
+            </div>
+          </div>
+        </div>
+        <div class="project-task-list">
+          <h2>Contribute</h2>
+          <div class="project-task">
+            <h3>Extract data</h3>
+            <div class="project-task-meta-information">
+              <p class="project-meta">Spreadsheets / Data interpretation</p>
+              <p class="project-meta">60 minutes+ / Laptop or desktop</p>
+            </div>
+            <p>Task description Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla eleifend vestibulum neque at blandit. Aliquam ac rhoncus felis. Etiam scelerisque viverra condimentum. Suspendisse faucibus lectus vel magna pretium, a imperdiet lorem aliquam. Vestibulum turpis eros.</p>
+            <div class="project-workload">
+              <h4 class="project-workload-title">Workload</h4>
+              <div class="project-workload-diagram">
+                <progress max="100" value="10" class="project-workload-indicator"></progress><span class="project-workload-value">100%</span>
+              </div>
+            </div>
+            
+            <a href="#" class="button">Start</a>
+          </div>
+        </div>
+      </div>
+      <aside class="project-aside">
+        <div class="project-owner">
+          <h2>Project owner</h2>
+          <img src="https://pbs.twimg.com/profile_images/1218922930018623488/C4ocKWB6_400x400.jpg" alt="" class="project-owner__photo" />
```

Comment:
This should be replaced with the actual code that was added:

```
<% if @project.owner.show_profile_photo? %>
  <%= image_tag get_profile_photo_url(url_name: @project.owner.url_name) %>
<% end %>
```

See the [`image_tag`](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_tag) API docs for adding HTML attributes.

Reply:
Done